src/fsharp/FSharp.Core/array.fs

```diff
@@ -57,7 +57,7 @@ namespace Microsoft.FSharp.Collections
         let create (count:int) (x:'T) =
             if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
             let array: 'T[] = Array.zeroCreateUnchecked count
-            for i = 0 to Operators.Checked.(-) count 1 do // use checked arithmetic here to satisfy FxCop
+            for i = 0 to Operators.Checked.(-) array.Length 1 do // use checked arithmetic here to satisfy FxCop
```

Comment:
Just checked on another function, no performance impact there either.   I'd be happy to get rid of all the Checked bits for readability purposes in this PR, or I could do another one. @KevinRansom  or @dsyme  any input here?